chef/cookbooks/cpe_chrome/resources/cpe_chrome.rb

```diff
@@ -94,5 +96,31 @@ def manage_chrome_osx
         ],
       }
     end
+
+    unless node['cpe_chrome']['mp']['UseMasterPreferences'] == false
+      mprefs = node['cpe_chrome']['mp']['FileContents'].reject { |v| v.nil? }
```

Comment:
I tested this and it does not work.

`Undefined method or attribute 'compact' on 'node'`

I think we can leave it as is and just reject the values.